website/cue/reference/components/sinks/splunk_hec_logs.cue

```diff
@@ -79,6 +79,24 @@ components: sinks: splunk_hec_logs: {
 				examples: ["https://http-inputs-hec.splunkcloud.com", "https://hec.splunk.com:8088", "http://example.com"]
 			}
 		}
+		endpoint_target: {
+			common:      false
+			description: """
+                                     The Splunk endpoint to send to. Either the [event endpoint](\(urls.splunk_hec_event_endpoint)) or the
+                                     [raw endpoint](\(urls.splunk_hec_raw_endpoint)). [metadata](\(urls.splunk_hec_metadata)) for the event
+                                     endpoint is sent with each event. For the raw endpoint it is sent via the url parameters.
```

Comment:
I think this needs to be updated now that we dropped `metadata` as a config option.

Reply:
This is my update following that change. It explains that the metadata (the fields described if you follow the `splunk_hec_metadata` link) are sent differently depending on which endpoint you choose.

Is it not worded clearly enough?

Reply:
Ah, yeah, I see. I might reword the third statement as:

```
For the `raw` endpoint, configured [event metadata](\(urls.splunk_hec_metadata)) is sent as query parameters, except the `timestamp` field.
```